tests/README.md

```diff
@@ -41,3 +41,7 @@ any entry (`transports/docker`) in the policy.json. Now it refers to
 repository `quay.io/kata-containers/confidential-containers` concretely.
 * `allow/deny`: Whether the image to be pulled in this test should be allowed
 or denied.
+
+In `signature_verification.rs`, the tests are organized due different kinds
+of KBCs, which means for each given KBC, all the 4 tests mentioned will be
```

Comment:
Nit: Numbers less than ten are better written as words:

```suggestion
of KBCs, which means for each given KBC, all four tests mentioned will be
```

Reply:
Thanks, sorry for non-native English user :-)